airflow/providers/amazon/aws/operators/s3_to_redshift.py

```diff
@@ -15,13 +15,20 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+import warnings
 from typing import List, Optional, Union
 
 from airflow.models import BaseOperator
 from airflow.providers.amazon.aws.hooks.s3 import S3Hook
-from airflow.providers.postgres.hooks.postgres import PostgresHook
 from airflow.utils.decorators import apply_defaults
 
+try:
+    from airflow.providers.postgres.hooks.postgres import PostgresHook
+except ImportError:
+    warnings.warn("The package providers-amazon-aws must be installed to use the operator.",
```

Comment:
I am not sure I understand this warning... should it be `postgres` instead of `provider-amazon-aws`?

Reply:
Yeah, It was manually copy-pasted and it's yet something to be reviewed. ... I have not fixed all of these and I want to change it to be more difficult to make copy-paste mistake